superset-frontend/src/components/Chart/Chart.tsx

```diff
@@ -150,9 +153,16 @@ const MonospaceDiv = styled.div`
   overflow-x: auto;
   white-space: pre-wrap;
 `;
+class Chart extends React.PureComponent<ChartProps, {}> {
+  static defaultProps = defaultProps;
+
+  renderStartTime: any;
```

Comment:
Where does `renderStartTime` come from? In the old version, we're only reading it, but we don't assign any value to it. In your changes you put `renderStartTime` in ChartProps, but also as class's field. I don't think it's actually passed as a prop anywhere, and as a class field it's also only read, not assigned.
So maybe we should just remove it?

Reply:
I removed this from ChartProps. 
You mean we should remove him from line 198 and 200 ( start_offset and duration keys)?

Reply:
Yeah, unless I misunderstand something here, renderStartTime is always undefined because we don’t set its value anywhere, so maybe we could remove it from the code. @rusackas any objections?

Reply:
Unless we want to actually fix it and assign a timestamp

Reply:
Yes, I think we should understand why it was put there.
I remain waiting for instructions, thank you.

Reply:
Well, if it's not wired up, that's an odd problem. It sure seems like it'd be useful for logging, which seems to be its intent. There's one in `ChartRenderer.jsx` that IS wired up, for reference. Maybe we don't need both though? 

It WAS set in the [original commit](https://github.com/apache/superset/commit/5c02e3199b924cc8a45622e660096d6fdc8c5857), actually. I haven't checked to see when its initialization was _removed_ but clearly there wasn't much outcry.

I'd err on the side of more logging, not less. Maybe @eschutho would be interested in taking more advantage of this insight :)

Reply:
We can prob remove it and add it back in later if we find that we need it.